frontend/src/app/app-routing.module.ts

```diff
@@ -1,15 +1,23 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
-
+import { AuthGuard } from './core/guards/auth.guard';
 import { LandingPageComponent } from './shell/landing-page/landing-page.component';
 
 const routes: Routes = [
   { path: '', component: LandingPageComponent },
-  { path: '**', redirectTo: '', pathMatch: 'full'}
+  { path: '**', redirectTo: '', pathMatch: 'full' },
```

Comment:
The last change. Could you move ** route to the end? It another way may work incorrectly